docs/3-Quickstart.md

```diff
@@ -1,16 +1,20 @@
 # Quickstart
 
-Once you have your API keys, you can start interacting with Finch's API using our sandbox mode — a test environment with life-like data.
+Once you have an application's `client_id` and `client_secret`, you can start interacting with Finch's API using our sandbox mode — a test environment with mock data.
 
-This guide will help you send your first request to Finch's API while the next guides dive deeper into the concepts and help you integrate Finch into your production application.    
+This guide will help you send your first request to Finch's API while the following guides dive deeper into the concepts and help you integrate Finch into your production application.
 
 ---
 
 ### Open Finch Connect in Sandbox mode
 
 Finch Connect provides a secure and elegant authorization flow for your users to grant your application access to their systems.
 
-Launch Connect by constructing and navigating to the following URL on your browser.
+> Note: this quickstart guide is a simplified, but manual way of generating an authorization `code` and exchanging it for an `access_token`, which can be used to subsequently call our APIs. In a true production environment, you will want to automate this process completely inside your application's code.
+
+Since this quickstart assumes you have not built an application yet, we must make sure that a proper `redirect_uri` is set up before continuing or our authorization code generation will fail. In your [Finch Dashboard](https://dashboard.tryfinch.com), go to the "Redirect URIs" section and select `+ Add Redirect URI`. We are going to use [https://example.com](https://example.com) for testing purposes. In production, you will want to use your own application's urls for the Redirect Uris (and remove all mentions of [https://example.com](https://example.com) or [http://localhost](http://localhost)). You must do this if you are [redirecting to Finch Connect](./Integrating-with-Finch/Integrate-Finch-Connect/Redirect-to-Connect.md), but you can use our [embedded Finch Connect flow](./Integrating-with-Finch/Integrate-Finch-Connect/Embed-Connect.md) to avoid any redirection. Currently, Finch only supports embedded flow with our [React SDK](https://developer.tryfinch.com/docs/guides/ZG9jOjEzNDk2NDE-react-tutorial)
```

Comment:
One word of caution here: If customers assume they need to use this same Redirect URI on their backend when converting to an access token, and they also use the React SDK, they will get a non-intuitive error about the auth token being invalid. The React SDK currently hardcodes `https://tryfinch.com` when it creates the auth token, and the backend must match.

Short version: Make sure to set the expectation that Redirect URIs are currently only used for URLs and not the React SDK.

(To some extent you've done this, just asking that it be kept in mind.)